README.md

```diff
@@ -1217,6 +1217,56 @@ const MyElement = (props, chren) => <div>Hello, {chren()}!</div>;
 
 
 
+#### `vdom.elements (...elements)`
+
+Returns an array of functions that create elements.
+
+```js
+const [ div, p ] = skate.vdom.elements('div', 'p');
+div(p.bind('test'));
```

Comment:
> I'm skeptical it would matter enough to warrant a poorer API. I'd want to see numbers on the perf implications before deciding against it.

My real concern is memory and negating the benefits of IncrementalDOM. That said, you're right: innocent until proven guilty.